sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -27,8 +27,14 @@
 if TYPE_CHECKING:
     from datetime import datetime
 
-def generate_sas(endpoint, shared_access_key, expiration_date_utc, **kwargs):
-    # type: (str, str, datetime, Any) -> str
+def generate_sas( # pylint: disable=unused-argument
+    endpoint: str,
+    shared_access_key: str,
+    expiration_date_utc: "datetime",
+    *,
+    api_version: str = constants.DEFAULT_API_VERSION,
+    **kwargs: Any
```

Comment:
we can remove the `**kwargs` here and then the pylint disable. We have typed out the kwargs we need here :)